src/components/Widgets/Markdown/MarkdownControl/Toolbar/Toolbar.js

```diff
@@ -47,31 +43,47 @@ export default class Toolbar extends React.Component {
 
   render() {
     const {
-      onH1,
-      onH2,
-      onBold,
-      onItalic,
-      onLink,
       onToggleMode,
       rawMode,
       plugins,
       onAddAsset,
       onRemoveAsset,
       getAsset,
+      disabled,
     } = this.props;
 
+    const buttons = this.props.buttons || {};
+
     const { activePlugin } = this.state;
 
+    const buttonsConfig = [
```

Comment:
That wasn't a thought when I did this, honestly. I'll probably refactor before the merge.